scripts/rnaseq2ga.py

```diff
@@ -290,4 +266,21 @@ def main(argv):
 
 
 if __name__ == '__main__':
-    main(sys.argv)
+    parser = argparse.ArgumentParser(
+        description="Script to generate SQLite database corresponding to "
+        "input RNA Quantification experiment files.")
+    parser.add_argument(
+        "--outputFile", "-o", default="rnaseq.db",
+        help="The file to output the server-ready database to.")
+    parser.add_argument(
+        "--inputDir", "-i",
+        help="Path to input directory containing RNA quant files.",
+        default='.')
+    parser.add_argument(
+        "--controlFile", "-c",
+        help="Name of control file (.tsv format) in the inputDir",
+        default="rna_control_file.tsv")
+    parser.add_argument('--verbose', '-v', action='count', default=0)
+    args = parser.parse_args()
+
+    rnaseq2ga(args.inputDir, args.outputFile, args.controlFile)
```

Comment:
I see why you have this here, but I think it's cleaner to leave the `main()` and `makeParser()` functions as they are (with the additions here moved to `makeParser()` of course) and then have the revised `main()` call the `rnaseq2ga()` function